api/handlers/auth.go

```diff
@@ -15,6 +15,71 @@ import (
 	"github.com/go-chi/render"
 )
 
+func (h *Handler) InitLoginSSO(w http.ResponseWriter, r *http.Request) {
+
+	configuration, err := config.Get()
+	if err != nil {
+		_ = render.Render(w, r, util.NewErrorResponse(err.Error(), http.StatusBadRequest))
+		return
+	}
```

Comment:
The handler already has a copy of the configuration. It should be h.A.Cfg

Reply:
Noted, thank you.